mlrun/launcher/remote.py

```diff
@@ -180,3 +181,14 @@ def _submit_job(
             resp = runtime._get_db_run(run)
 
         return self._wrap_run_result(runtime, resp, run, schedule=schedule)
+
+    @classmethod
+    def _validate_run_single_param(cls, param_name, param_value):
+        if isinstance(param_value, pd.DataFrame):
+            raise mlrun.errors.MLRunInvalidArgumentTypeError(
+                f"Parameter '{param_name}' has an unsupported value of type"
+                f" 'pandas.DataFrame' in remote execution. Top 3 records: \n{param_value.head(3)}\n"
```

Comment:
```suggestion
                f" 'pandas.DataFrame' in remote execution."
```
Printing the top 3 lines of the DF is way too much logs. The user doesn't really need this to identify what parameter this is, since you give the param name in the error, and all it does is clutter the logs.